jsk_2020_10_semi/launch/auto_temp_checker.launch

```diff
@@ -0,0 +1,41 @@
+<launch>
+  <arg name="launch_pointcloud_screenpoint" value="false" />
+  <!-- subscribe /head_camera/rgb/image_rect_color/screenpoint and publish /tf (ray_target) -->
+  <include file="$(find jsk_2020_10_semi)/launch/pointcloud_screenpoint.launch"
+           if="$(arg launch_pointcloud_screenpoint)" >
+    <arg name="machine" value="fetch1075" />
+  </include>
+  <include file="$(find jsk_2020_10_semi)/launch/image_view2.launch" >
+    <arg name="machine" value="fetch1075" />
+  </include>
+
+  <!-- publish /head_camera/rgb/image_rect_color/screenpoint from /edgetpu_human_pose_estimator/output/poses  -->
+  <node name="aws_auto_checkin_app" pkg="jsk_perception"
+        type="aws_auto_checkin_app.py"
+        ns="/head_camera/rgb"
+        output="screen" >
+    <remap from="image" to="image_rect_color" />
+    <param name="use_window" value="true" />
+    <param name="env_path" value="/tmp/env.json" />
```

Comment:
@708yamaguchi 
this requires https://github.com/jsk-ros-pkg/jsk_recognition/pull/2583,
and we need env file, download from
https://github.com/k-okada/jsk_recognition/blob/949a39120dfa2b52456e464b23f70b323bd83f23/doc/jsk_perception/nodes/aws_auto_checkin_app.rst#L78
Where should we put this file? `/etc/ros/` somewhere?